lib/car.rb

```diff
@@ -0,0 +1,24 @@
+class Car
+  attr_reader :make, :model, :monthly_payment, :loan_length, :total_cost
+  def initialize(make, model, monthly_payment, loan_length)
```

Comment:
The interaction pattern shows 3 arguments, you have 4 -- it is very important that you follow the patterns exactly to the letter.

Reply:
Note that you CAN actually perform logic inside of your `initialize` methods - the idea here is to take one argument of both make and model, and then split them via `.split(" ")`, a Ruby method that can separate parts of a string into an array.